website/docs/integrator-section/transferable-record/preparing-transferable-record.md

```diff
@@ -3,36 +3,37 @@ id: preparing-transferable-record
 title: Preparing Transferable Record
 sidebar_label: Preparing Transferable Record
 ---
+import WrapSingleDocument from "/src/reusables/_wrap-single-document.mdx";
 
 In this section, we will prepare the DNS and the content of the transferable record to be issued.
 
-## DNS Configuration
+## DNS configuration
 
 Similar to [binding the document store to a domain name](/docs/integrator-section/verifiable-document/ethereum/document-store), you will have to bind the identity of the token registry to a domain name.
 
-To do that simply create a `TXT` record on your domain with the following entry:
+To do that, create a `TXT` record on your domain with the following entry value:
 
-```txt
-openatts net=ethereum netId=3 addr=0x8431012Bc040942B59e3C5bf428221eab0b2f723
-```
+| Type | Domain      | Entry Value                                                     |
+| ---- | ----------- | --------------------------------------------------------------- |
+|`TXT` | example.com | "openatts net=`ethereum` netId=`11155111` addr=`0x8431012Bc040942B59e3C5bf428221eab0b2f723`" |
 
-You will need to replace the token registry address `0x8431012Bc040942B59e3C5bf428221eab0b2f723` with the address you've got from the previous step.
+You will need to replace the token registry address `0x8431012Bc040942B59e3C5bf428221eab0b2f723` with the address you get from the previous step.
 
-If you like to use our sandbox DNS for the purpose of the exercise, you may instead run the following command, following the instructions from [earlier](/docs/integrator-section/verifiable-document/ethereum/dns-proof):
+If you want to use our sandbox DNS for the purpose of the exercise, run the following command instead following the instructions from [earlier](/docs/integrator-section/verifiable-document/ethereum/dns-proof):
 
 ```sh
-open-attestation dns txt-record create --address 0x8431012Bc040942B59e3C5bf428221eab0b2f723 --network-id 3
+open-attestation dns txt-record create --address 0x8431012Bc040942B59e3C5bf428221eab0b2f723 --network-id 11155111
 ```
 
-If you like more detailed setup instructions, you may refer to the [documentation for configuring DNS](/docs/developer-section/quickstart/configure-dns).
+If you want to view more detailed setup instructions, see the [documentation for configuring DNS](/docs/developer-section/quickstart/configure-dns).
 
-> Take note of the domain you are inserting the records on, you will need this later
+>**Important:** Take note of the domain where you are inserting the records, because you will need the domain name later.
 
-## Creating Raw Transferable Document
+## Creating a raw transferable document
 
 Similar to creating a verifiable document, you will need to create a raw JSON file with the content of the transferable record first.
 
-Create a file `sample.json` in a folder `raw-documents`:
+Create a file `sample.json` in a folder `raw-tr`:
```

Comment:
In the previous sections we name the folders like `raw-documents` and `raw-documents-did`. Should we follow this and use `raw-documents-tr`or `raw-documents-transferable`  for this section? 

Or is there some reason `raw-tr` was used? Same for the wrapped and signed document folders.

Reply:
`raw-tr` stands for "raw transferable record". But it looks confusing. 
Changed `raw-tr` to `raw-documents-transferable`.
Changed `wrapped-tr` to `wrapped-documents-transferable`.